### ISSUE TITLE
Mavan install produces 2 target folders

### ISSUE THREAD

**Issue by evelinec** [OpenLiberty/guide-rest-intro#65]
The pom.xml file may have some problem. There are 2 target folders being produced, one with corrupted files. 

evelines-mbp:start evelinec$ ls
pom.xml						target
src						target?                                 
evelines-mb

**Comment by andrewdes** [comment]
This issue is due to the following format change in the maven pom.xml file:
```
                <configuration>
                    <summaryFile>${project.build.directory}
                                 /test-reports/it/failsafe-summary.xml</summaryFile>
                    <reportsDirectory>${project.build.directory}/test-reports/it
                                    </reportsDirectory>
                </configuration>
```

**Comment by andrewdes** [comment]
I opened a PR here: https://github.com/OpenLiberty/guide-rest-intro/pull/66/files

**Comment by gkwan-ibm** [comment]
fixed by PR #66
